autoPyTorch/pipeline/components/setup/network_backbone/ResNetBackbone.py

```diff
@@ -353,8 +341,9 @@ def forward(self, x: torch.FloatTensor) -> torch.FloatTensor:
                     x.is_cuda,
                 )
                 x = shake_drop(x, alpha, beta, bl)
+            else:
+                x = self.layers(x)
 
-        if self.config["use_skip_connection"]:
```

Comment:
why are you removing this here? residual will not exist if there is no skip connection see line 309.

Reply:
Hey @ravinkohli, it is inside of the skip connection condition now https://github.com/automl/Auto-PyTorch/blob/3b7f5598b32ce4614d18605f18e00b05fa89b65a/autoPyTorch/pipeline/components/setup/network_backbone/ResNetBackbone.py#L327 
Maybe just looking at the changes gives the wrong view, however, you can look at the file and let me know if you see a bug again.

Reply:
yes thats true. Looking at the whole file I think these changes are fine. Thank you